### PR TITLE
Changed previous upstream name to 'Internal' channel name.

### PR DESCRIPTION
**Pull request by pboguslawski**
## Proposed change
This mod changes internal channel name from "OTRS" to "System" to avoid using previous upstream name.

## Type of change
- '1 - 🐞 bug 🐞'            - Bugfix (non-breaking change which fixes an issue)

## Additional information

Author-Change-Id: IB#1108954

## Checklist

- [x] The code change is tested and works locally.(❗)
- [x] There is no commented out code in this PR.(❕)
- [ ] You improved or added new unit tests.(❕)
- [x] Local ZnunyCodePolicy run passes successfully.(❕)
- [x] Local unit tests pass.(❕)
- [x] GitHub workflow ZnunyCodePolicy passes.(❗)
- [x] GitHub workflow unit tests pass.(❗)

### CODE REVIEW

**Comment by dennykorsukewitz** [comment]
Hi @rkaldung ,

I like `System` instead of Znuny.
What do you think?

**Comment by rkaldung** [comment]
> I like `System` instead of Znuny. What do you think?

I would avoid System, it sounds like sth. automatic to me. Do we have a summary of where this channel is used?

File: Kernel/Config/Files/XML/Framework.xml [hunk 1]
```diff
@@ -8851,7 +8851,7 @@ via the Preferences button after logging in.
         <Navigation>Core::CommunicationChannel</Navigation>
         <Value>
             <Hash>
-                <Item Key="Name" Translatable="1">OTRS</Item>
+                <Item Key="Name" Translatable="1">System</Item>
```
Comment (dennykorsukewitz) [inline Kernel/Config/Files/XML/Framework.xml]: We want to use the "original" backend name.

```suggestion
                <Item Key="Name" Translatable="1">Internal</Item>
```

**Review by rkaldung — COMMENTED** [review]
@dennykorsukewitz We'll keep it Internal then?

**Comment by dennykorsukewitz** [comment]
@rkaldung Jap `Internal`

**Review by rkaldung — APPROVED** [review]
`Internal` approved

**Review by dennykorsukewitz — APPROVED** [review]

**Pull request merged**